TinyConstraints/Classes/Stack.swift

```diff
@@ -28,7 +28,7 @@
     import UIKit
 #endif
 
-public extension TinyView {
+@MainActor public extension TinyView {
```

Comment:
Not needed since `TinyView` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```